BricksAndHearts.UnitTests/ControllerTests/Landlord/LandlordControllerTests.cs

```diff
@@ -81,10 +83,11 @@ public async void ApproveCharter_WithoutMembershipId_DisplaysErrorMessage()
 
         // Act
         await UnderTest.ApproveCharter(landlordProfile);
+        var flashMessages = UnderTest.TempData["FlashMessages"] as List<string>;

```

Comment:
As we are calling the same thing everytime to access the flashMessages, we could add something to the TestsBase, something like 
```
protected IEnumerable<string>? FlashMessages => UnderTest.TempData["FlashMessages"] as List<string>;
```

This would give us access to the FlashMessages without having to add them into all the Acts